freqtrade/optimize/hyperopt_tools.py

```diff
@@ -298,7 +299,7 @@ def _format_explanation_string(results, total_epochs) -> str:
                 f"Objective: {results['loss']:.5f}")
 
     @staticmethod
-    def prepare_trials_columns(trials, legacy_mode: bool, has_drawdown: bool) -> str:
+    def prepare_trials_columns(trials, legacy_mode: bool, has_drawdown: bool) -> pd.DataFrame:
```

Comment:
I think you should also type "trials" then ... to make it complete (that's why it wasn't captured as part of mypy in the first place).

Reply:
Hmm, not sure I understand. Where would I type "trials"?

Reply:
What i mean is type the parameter (as follows): 

`def prepare_trials_columns(trials: pd.DataFrame, ...`

Reply:
Ah yes! Good point :)